src/query-compiler/default-query-compiler.ts

```diff
@@ -1509,6 +1515,12 @@ export class DefaultQueryCompiler
     }
   }
 
+  protected override visitFetch(node: FetchNode): void {
+    this.append('fetch next ')
+    this.visitNode(node.rowCount)
+    this.append(` rows ${node.modifier}`)
```

Comment:
SQL injection with modifier. Not very serious one. We might have these for modifiers in the existing code too. But we should start checking the values for everything that gets interpolated to the SQL.

Reply:
Great catch!